ee/modules/160-multitenancy-manager/templates/user-resources/_helpers.tpl

```diff
@@ -0,0 +1,6 @@
+{{- define "slugify" }}
```

Comment:
Honestly, I'm a bit confused here.
With this change we will have the `slugify` func defined twice, here and [here](https://github.com/deckhouse/deckhouse/blob/main/ee/modules/160-multitenancy-manager/templates/user-resources/templates/user-resources-templates.yaml#L15)

Also, according to [this code](https://github.com/deckhouse/deckhouse/blob/main/ee/modules/160-multitenancy-manager/hooks/project.go#L143-L168) module will load files only from the `templates/` dir and this file will not be loaded. Probably, you have to add this function to [this file](https://github.com/deckhouse/deckhouse/blob/main/ee/modules/160-multitenancy-manager/templates/user-resources/templates/_helpers.tpl) but anyway we have to fix the code duplication.


I think the plan should be like this:
1.  Move [this func](https://github.com/deckhouse/deckhouse/blob/main/ee/modules/160-multitenancy-manager/templates/user-resources/templates/user-resources-templates.yaml#L15-L39) to the `templates/_helpers.tpl` file
2. Simplify/adopt it (if necessary)
3. Use it in templates and also [here](https://github.com/deckhouse/deckhouse/blob/main/ee/modules/160-multitenancy-manager/templates/user-resources/templates/user-resources-templates.yaml#L106)

Reply:
Thank you! I completely agree with you.

Made changes, transferred templates to one file. But I didn’t use `slugify` myself, since it uses two parameters. Renamed my function to `normalize`.

Conducted testing, helm charts were generated correctly.